src/kit/LineChart.tsx

```diff
@@ -260,7 +260,7 @@ export const LineChart: React.FC<LineChartProps> = ({
   ]);
 
   return (
-    <div className="diamond-cursor">
+    <Surface>
```

Comment:
all card shave surface applied to them now -- is that okay?

Reply:
yeah, my comment was re: an example with Cards in *another* Surface so that we can see the higher elevation (like examples in List and Pivot sections). don't think it's critical though.